client/my-sites/checkout/src/components/payment-method-step.tsx

```diff
@@ -67,9 +68,28 @@ const NonTotalPrices = styled.div`
 `;
 const TotalPrice = styled.div`
 	font-size: 14px;
-	padding: 16px 0;
+	padding-top: 16px;
```

Comment:
I'm going to add back a little bit of bottom padding here.  Changing it to 0 made the new text bunch together too much with the "Total" line above it.